source/date_time.md

```diff
@@ -0,0 +1,89 @@
+# Date and time question types
+
+There are 3 different date and time question types in KoboToolbox: “Date”,
+“Time” and “Date and time”.
+
+The “Date” question type is for capturing date values, for example on a question
+like “What is your date of birth?” In both KoboCollect and Enketo web forms, a
+calendar-style date picker will be shown for selecting the date.
+
+The “Time” question type is for capturing time values, for example on a question
+like “At what time do you leave for work?” In both KoboCollect and Enketo, a
+time picker is shown where a user can select their response.
+
+The third type “Date and time” is for capturing both date and time responses on
+a single question.
+
+## How to set up Date and Time question types
+
+### Setting up in FormBuilder
+
+Adding “Date”, “Time”, and “Date and Time” questions is simple and does not
+require any extra setup:
+
+-   In the KoboToolbox Formbuilder, click the + button to add a new question
+-   Type the question text, for example “What is your date of birth?”, then
+    click ADD QUESTION or press ENTER on your keyboard
+-   Choose the question type
+
+![Adding the questions](images/date_time/adding.gif)
+
+### Setting up in XLSForm
+
+To add date, time, and date and time questions in the XLSForm, use the date,
+time and dateTime question types as shown in the example below:
+
+In XLSForm, you can set up the following:
+
+| type     | name      | label                                         |
+| :------- | :-------- | :-------------------------------------------- |
+| date     | dob       | On what date were you born?                   |
+| time     | time      | At what time do you you leave for work?       |
+| dateTime | date_time | At what date and time did the training start? |
+
+## Appearance of date and time question types in web forms and KoboCollect
+
+### Default appearance
+
+![Default appearances](images/date_time/default_appearances.png)
+
+### Advanced appearances
+
+When adding the “Date” question type, you can choose from a number of display
+options (under the question settings). Appearances change the way the question
+is displayed on web forms and on KoboCollect.
+
+![Default appearances](images/date_time/advanced_appearance.png)
+
+![Default appearances](images/date_time/advanced_appearances.png)
+
+### Adding custom appearances for date questions in XLSForm
+
+You can specify advanced appearances in XLSForm through the appearance column as
+follows:
+
+| type | name         | label                              | appearance |
+| :--- | :----------- | :--------------------------------- | :--------- |
+| date | rains_start  | When did the planting rains start? | month-year |
+| date | year_migrate | In which year did you migrate?     | year       |
```

Comment:
Note that there is also a third calendar appearance of `no-calendar`. Since you've already mentioned two, you might as well specify all three. Also make sure to note that there are different non-Gregorian calendars that can be used, as listed [here](https://docs.getodk.org/form-question-types/#id172) in the ODK docs.

Reply:
Done